README.md

```diff
@@ -243,6 +243,11 @@ This is a custom hook that takes care of fetching your query and storing the res
     - `previousData`: Previous GraphQL query or `updateData` result
     - `data`: New GraphQL query result
   - `client`: GraphQLClient - If a GraphQLClient is explicitly passed as an option, then it will be used instead of the client from the `ClientContext`.
+  - `refetchAfterMutations`: list of mutations that will trigger the query refetch. It can be either:
+    - String | String[] - The string containing the mutation's definition or an array of them
+    - Object | Object[] with the following properties or an array of them
+      - `mutation`: String - The mutation string
+      - `filter`: Function (optional) - It receives mutation variables as parameter and blocks refetch if it returns false
```

Comment:
I would consider describing it in a different way:

- avoid mentioning "list of mutations", because in fact it could be only one
- the type definition should actually be `string | object | (string | object)[]`. Then, once you define the type you describe
  - if it's a string, it's the mutation string
  - if it's an object then it has properties mutation and filter
  - if it's an array, the elements can be of either type above

rephrase in a way that makes sense to the reader :)